browser/src/CopyNumberVariantPage/__snapshots__/CopyNumberVariantPage.spec.tsx.snap

```diff
@@ -779,7 +779,7 @@ exports[`CopyNumberVariantPage with dataset gnomad_cnv_r4 with variant of type D
   </div>
   <section>
     <h2>
-      Population Frequencies
+      Genetic ancestry group Frequencies
```

Comment:
All section titles (like on variant pages) should be in title case (Genetic Ancestry Group Frequencies)